docs/pages/_bundles/2CamelConfig.md

```diff
@@ -7,16 +7,84 @@ permalink: bundles/camelconfig
 # CamelConfig
 
 ## The goal of the bundle
-CamelConfig is a bundle designed to parse the Apache Camel xml flows and translate this into Symfony services.
+CamelConfig is a bundle designed to parse the Apache Camel XML flows and translate this **into Symfony services**.
 
 ## Main features
+
+* Easy to use and implement.
+* Route versioning.
+* Full support for complex routing, like *multicasts* or *recipient lists*.
+* Clear route definition, based **on Apache's specification**.
+* Fully compatible **with Symfony 2.8**. (Compatibility with 3.4 is part of [the roadmap](smartesb-skeleton/roadmap))
+* Battle tested.
+
 ## Installation
+Installation is as easy as installing a new bundle in your Symfony application. First start by running `composer`:
+
+`$ composer require smartbox/camel-config-bundle`
+
+Then you need to **enable the bundle** in your kernel.
+
+```php
+// app/AppKernel.php
+
+class AppKernel extends Kernel
+{
+    public function registerBundles()
+    {
+        $bundles = [(
+            new Smartbox\Integration\CamelConfigBundle\SmartboxIntegrationCamelConfigBundle(),
+            (...)
+```
+
+And you're all set.
+
 ## Usages
 ## How FlowsBuilder compiler pass works
-To see more details on how the ```FlowsBuilder``` compiler pass works click here [here](/smartesb-skeleton/camel/flows-builder)
+To see more details on how the ```FlowsBuilder``` compiler pass works click [here](/smartesb-skeleton/camel/flows-builder)
 
 ## How to build a flow
-## Use eclipse to build a flow
+
+For asynchronous flows follow [this guide](/smartesb-skeleton/samples/asyncflow), for synchronous, [this one](/smartesb-skeleton/samples/syncflow).
+
+## Use Eclipse to build a flow
+
+[JBoss Tools](http://marketplace.eclipse.org/content/jboss-tools) is a plugin for Eclipse that can aid you visually in the process of designing Apache Camel flows. 
+
+After installing it from Eclipse's Marketplace you can create a new file by selecting `Red Hat Fuse/Camel XML File`. This will create a base file but you can get creative **by dragging and dropping** the various components of the routes.
+
+![Sample Camel Routing](/smartesb-skeleton/assets/images/camel_routing_sample.jpg)
+
+To create a simple direct routing, just drag two `Direct` components one after the other. Eclipse **will join them for you**. Clicking on the components and opening the properties view will allow you to edit the uri, description and ID.
+
+![Camel Route Direct](/smartesb-skeleton/assets/images/camel_single_route.jpg)
+
+Once you're happy with the flow, you can export it by copying and pasting the source code present **in the "Source" tab**.
```

Comment:
In this case instead of mentioning copy and paste I would say to save the file

Reply:
And also we could relate this part of the of how the xml of the flows is translated to the processors

Reply:
Good point, fixed